src/string.cr

```diff
@@ -489,13 +519,14 @@ class String
   end
 
   # :nodoc:
-  record ToU64Info,
-    value : UInt64,
-    negative : Bool,
-    invalid : Bool
+  record UIntInfo, value : (UInt64 | UInt128), negative : Bool, invalid : Bool
 
   private macro gen_to_(method, max_positive = nil, max_negative = nil)
-    info = to_u64_info(base, whitespace, underscore, prefix, strict, leading_zero_is_octal, unsigned: {{max_negative == nil}})
+    if base == UInt128
+      info = to_info(base, whitespace, underscore, prefix, strict, leading_zero_is_octal, unsigned: {{max_negative == nil}})
+    else
+      info = to_info(base, whitespace, underscore, prefix, strict, leading_zero_is_octal, unsigned: {{max_negative == nil}})
```

Comment:
What's the reason for having two branches doing the same thing?

Reply:
I thought this was removed.  It is legacy from when I started this branch.